### DISCUSSION TITLE
Illa is crushing when I am trying to expand a field

### DISCUSSION THREAD

**Discussion by dianaspetcu** [illacloud/illa-builder#2894] [English Support]
Hi team. I have an issue. why the illa is crushing when i am trying to expand a field? it is not the first time.... it is very annoying.

![image](https://github.com/illacloud/illa-builder/assets/132670969/035eb226-6789-4f15-b746-9f98799d00fa)

**Comment by dianaspetcu** [comment]
I am trying to expand one of the fields from the modal, to be more larger and when I do that the illa give me the error that the page does not respond and i have to refresh the page

**Reply by donqu1xotechina** [reply]
Hey there:
I believe I DMed you on Discord, Could you please take a look when you have time?
